drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -19,6 +19,9 @@ function DragAndDropBlock(runtime, element, configuration) {
     var TAB = 9;
     var M = 77;
 
+    // Event string size limit.
+    var MAX_LENGTH = 255;
+
```

Comment:
@Kelketek This is still in the wrong place. Can you move it above the `// Keyboard accessibility` section, so that the latter is not broken up by this block?